content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,462 @@
+
+It is also possible to deactivate, in some cases, the default _higlight reconstruction_ module if _filmic_ reconstruct itself the highlights, to preserve more details. Then think of controlling the desaturation of the highlights in order to absorb the magenta highlights that you could create by deactivating this module.
+
+# Others improvements
+
+This 3.2, like the 3.0, is so rich in new features and improvements. This article doesn't fit to be exhaustive but to present the main changes and allow you to understand them. You will find below, in a jumble, a few other improvements to know.
+
+## Better management of the image processing pipeline
+
+As a reminder, darktable 3.0 has not only introduced a new processing pipeline order but also the possibility to change the order of this pipeline by reordering the modules yourself (very advanced function). In fact, some users got lost and had no way to find the default order of 3.0 or the one of 2.6.x and earlier versions.
+
+So, a new permanent module has been added in _darkroom_ (in bottom of right panel) named _module order_. This module allows to check the pipeline order actually applied to the current image: 3.0, original (default order on versions older than 3.0) and custom (personal order that you define, even unintentionally). This same module also allows you to apply 3.0 preset, or original one, but also to save your own custom preset. Those are the order you can find on new filter collections presented before in this article.
+
+## Performance improvements and important bug fixes
+
+It has been noticed that bugs, sometimes important, are present in version 3.0.x (up to 3.0.2). Especially bugs related to the display of images, especially in case of multiple instances (but not only). Most of these bugs resulted from the new image processing order (pipeline), integrated in version 3.0. A complete overhaul of the processing pipeline has been done by Pascal Obry for a more robust management of the new pipeline integrated in 3.0. This overhaul makes the pipeline both more stable and more efficient.
+
+Image modification detection has been improved. This mainly avoids the need to recalculate the display of an image that has not been altered, for example by having been opened in a darkroom without any modification. The modification of XMPs and thumbnails is thus more reliable and only impacted in the event of a real modification.
+
+Many other performance improvements have been integrated in this version 3.2, especially in some modules. This work is in progress since several other proposals for performance enhancements in other modules are already in progress for integration in the future 3.4. Thus, the next 3.4 seems to be shaping up with significant performance improvements.
```

Comment:
Consider removing the text ", especially in some modules"

Reply:
No, that's the case. Apart of lighttable overhaul, most performance improvements have been integrated especially in some modules actually.

Reply:
It just doesn't particularly read well when you use the word 'especially' (which in English reads as 'specifically') and then say 'some' modules (which isn't very specific). I considered that the easiest solution was to delete the words since it doesn't change the meaning of the sentence.

Reply:
That just don't precise than most of improvements are on modules.

Reply:
I could replace especially by "most"v or "mainly"

Reply:
How about "especially in image operation modules"